src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -875,8 +875,11 @@ export default {
             params: { correctedFilingId: item.filingId } })
           break
         case FilingTypes.INCORPORATION_APPLICATION:
-          // for now, not allowed
-          alert('At this time, you cannot correct an incorporation application. Please contact Ops if needed.')
+          // redirect to Correct web app to correct this Incorporation Application
+          const correctUrl = sessionStorage.getItem('CORRECT_URL')
+          const url = `${correctUrl}correction/?id=${item.filingId}`
```

Comment:
This will require a change in the framework to use the filingId opposed to the business identifier. I went that path, as i was thinking down the road to Alteration and Profile view as well. Maybe best for now to leave this as is, and alter the correction to take the filingId?

Reply:
Ah, right, good catch.
I guess it does make sense to open Correct UI with a filing in mind.
I will update the code above to refer to the parameter as "filingId" (so noone else confuses it with business id).

Reply:
You bet, will put in the small change to handle the Id in the correct-ui 👍 . It definitely does make sense for a correction!

Reply:
For anyone else reading this:
Cam and I decided that, when correcting an IA, the query param is "filingId".
However, when the Correct UI allows filing an alteration, or viewing/updating the business profile, then the query param will be "id" (or perhaps businessId).